packages/base/Breadcrumbs/src/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -1,21 +1,40 @@
 import type { ReactNode } from 'react'
 import React, { forwardRef } from 'react'
 import type { BaseProps } from '@toptal/picasso-shared'
-import { Breadcrumbs as MuiBreadcrumbs } from '@material-ui/core'
-import { ChevronRight16 } from '@toptal/picasso-icons'
-import './styles'
+import { Typography } from '@toptal/picasso-typography'
+
+import insertSeparator from './utils/insertSeparator'
 
 export interface Props extends BaseProps {
   /** Content of Breadcrumbs */
   children: ReactNode
 }
 
-export const Breadcrumbs = forwardRef<HTMLElement, Props>(function Breadcrumbs(
-  props,
-  ref
-) {
-  return <MuiBreadcrumbs ref={ref} separator={<ChevronRight16 />} {...props} />
-})
+export const Breadcrumbs = forwardRef<HTMLDivElement, Props>(
+  function Breadcrumbs(props, ref) {
+    const { children } = props
+
+    const allItems = React.Children.toArray(children).map((child, index) => (
+      // here is fine to use index as key (https://github.com/jsx-eslint/eslint-plugin-react/issues/1123)
+      // eslint-disable-next-line react/no-array-index-key
+      <li className='p-0 m-0' key={`child-${index}`}>
+        {child}
+      </li>
+    ))
+
+    return (
+      <Typography
+        as='nav'
+        ref={ref}
+        className='text-md text-black text-opacity-[0.54]'
```

Comment:
The new documentation for [text opacity mentions a slash syntax](https://tailwindcss.com/docs/text-color#changing-the-opacity), I believe we should use it as `text-opacity` is not present anymore on Tailwind v3 documentation
```suggestion
        className='text-md text-black/[.54]'
```

Reply:
nice! if the version that we use supports this, it looks even better